packages/sample-apps/windows/SampleAppCS/packages.lock.json

```diff
@@ -22,8 +22,8 @@
       },
       "ReactNative.Hermes.Windows": {
         "type": "Direct",
-        "requested": "[0.10.0-ms.1, )",
-        "resolved": "0.10.0-ms.1",
+        "requested": "[0.11.0-ms.2, )",
+        "resolved": "0.11.0-ms.2",
         "contentHash": "U+0FecIUFhmuLDNGT+kIhQdzoIgOvaGHU5gQOl66G5JdbO8xpful+E5OH/NL9Ue6N1l+ecqTH9shGrVDsdG5ow=="
```

Comment:
@JunielKatarn re the offline discussion about reproducible restoration, do you know why we're allowing incorrect package hash in our lockfiles? I know we used to be strict on these.